.github/workflows/dh-backend-ci.yml

```diff
@@ -37,18 +39,35 @@ on:
   pull_request:
     branches:
       - main
-    paths:
-      - 'build/infrastructure/dh/**'
-      - 'apps/dh/api-dh/**'
-      - '.github/workflows/dh-backend-ci.yml'
-      - '.github/actions/publish-release-contents/action.yml'
 
 jobs:
-  dotnet_solution_ci:
-    uses: Energinet-DataHub/.github/.github/workflows/dotnet-solution-ci.yml@v8
+  #
+  # Detect changes to start relevant jobs
+  #
+
+  changes:
+    uses: ./.github/workflows/dh-backend-detect-changes.yml
+
+  #
+  # CI .NET
+  #
+
+  # Build all projects within solution
+  dotnet_ci_build:
+    needs: changes
+    if: ${{ needs.changes.outputs.dotnet == 'true' }}
+    uses: Energinet-DataHub/.github/.github/workflows/dotnet-build-prerelease.yml@v9
     with:
       SOLUTION_FILE_PATH: 'apps/dh/api-dh/DataHub.WebApi.sln'
-      PREPARE_OUTPUTS: true
+
+  # Run all tests in 'Tests.dll'
+  dotnet_ci_test_unittests:
```

Comment:
These are not all unit tests, so maybe we should just call it 'dotnet_ci_test' until we add another test project